### PR TITLE
Cancel previous execution of PR github action

### PR DESCRIPTION
**Pull request by McGiverGim**
Cancel the previous execution of the CI for a PR, when the developer pushes new changes to the branch (new CI for the PR starts).

### CODE REVIEW

**Comment by haslinghuis** [comment]
https://stackoverflow.com/questions/74737407/what-is-concurrency-in-github-actions

**Review by haslinghuis — APPROVED** [review]

**Comment by McGiverGim** [comment]
> https://stackoverflow.com/questions/74737407/what-is-concurrency-in-github-actions

I think better the official source ;) https://docs.github.com/en/actions/using-jobs/using-concurrency

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by nerdCopter — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
